docerina-ui/src/component/fields.js

```diff
@@ -48,7 +49,8 @@ const Fields = (props) => {
                                 <ul>
                                     {item.type.memberTypes.map(type => {
                                         return(
-                                            <li>{type.name} {getTypeLabel(type.elementType, type.defaultValue)}
+                                            <li>{type.name} 
+                                            {/* {getTypeLabel(type.elementType, type.defaultValue)} */}
```

Comment:
Shall we remove the comments line

Reply:
This is disabled temporarily.
There are some improvements to be done in order to correctly show these types.